docs/tables/pagerduty_incident.md

```diff
@@ -0,0 +1,70 @@
+# Table: pagerduty_incident
+
+An incident represents a problem or an issue that needs to be addressed and resolved.
+
```

Comment:
We should include a line mentioning that by default only incidents from the last 30 days will be retrieved if `created_at` is not specified:

```
If no `created_at` key qual is specified, incidents from the last 30 days will be returned by default.
```